src/views/Requests/Communities.tsx

```diff
@@ -17,7 +17,7 @@ import String from '../../libs/Prismic/components/String';
 const Communities = ({ communities } : any) => {  
 
     return (
-        <Grid colSpan={1.5} cols={{ sm: 4, xs: 1 }}>
+        <Grid colSpan={1.5} cols={{ lg: 4, md:2, sm: 2, xs: 1 }}>
```

Comment:
You shouldn't need to declare the `md` since you're applying the size to `sm` and you just want to change when `lg`.